<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalFieldDialog</class>
 <widget class="QDialog" name="NormalFieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Normal Field Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Registar.qrc">
    <normaloff>:/images/bunny.png</normaloff>:/images/bunny.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Custom Viewpoint</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="XLineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="YLineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ZLineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Orient normal by custom viewpoint</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orient normal by viewpoint of visualizer camera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orient normal by viewpoint of scanning camera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Set normal directly towards custom viewpoint</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="defaultButton">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="overwriteCheckBox">
         <property name="text">
          <string>overwrite</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/Registar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>NormalFieldDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
